packages/redux-dynamic-modules-react/src/DynamicModuleLoader.tsx

```diff
@@ -100,12 +55,14 @@ class DynamicModuleLoaderImpl extends React.Component<
     private _addedModules?: IDynamicallyAddedModule;
     private _providerInitializationNeeded: boolean = false;

```

Comment:
We should add doc comments on each member of the interface as not all are self explanatory.